### PR TITLE
BD-1673 browndog shiny type

### PR DESCRIPTION
**Pull request by yan130**
BD-1673 browndog shiny type

## Description
use the list in register of data.atmosphere to get the 'type'

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: shiny/BrownDog/server.R [hunk 1]
```diff
@@ -8,10 +8,14 @@ library(PEcAn.visualization)
...
 server <- shinyServer(function(input, output, session) {
   output$typeSelector <- renderUI({
-    bety <- betyConnect("../../web/config.php")
-    con <- bety$con
-    on.exit(db.close(con))
-    datatype <- db.query("SELECT name FROM sitegroups;", con)
+    # get the files in library PEcAn.data.atmosphere under registration
+    libPaths<-.libPaths()
+    index <- dir.exists(paste0(libPaths, '/PEcAn.data.atmosphere'))
+    data.atmosphere.registration <-paste0(libPaths[index], "/PEcAn.data.atmosphere/registration")
```
Comment (robkooper) [inline shiny/BrownDog/server.R]: Use `system.file("registration", package="PEcAn.data.atmosphere")`

Comment (yan130) [inline shiny/BrownDog/server.R]: done. 

but it is "Ameriflux" & "Fluxnet2015". 

I should map them to AmeriFlux or change  agreement, to 

"Ameriflux" = checkboxInput("agreement", HTML("I agree to <a href='http://ameriflux.lbl.gov/data/data-policy/'>AmeriFlux license</a>."), value = FALSE, width = NULL)

?

Comment (robkooper) [inline shiny/BrownDog/server.R]: change agreement line

Comment (yan130) [inline shiny/BrownDog/server.R]: done

**Pull request merged**